packages/prompts/src/index.ts

```diff
@@ -624,3 +624,41 @@ Only return Node.js code. DO NOT include any English text before or after the No
 
 `;
 };
+
+export const englishStyleGuidePrompt = `# English Style Guide
+
+## Instructions
+
+Your goal is to apply two style guides to any English you find in the code:
+
+1. Google's Developer docs style guide (https://developers.google.com/style)
```

Comment:
```suggestion
1. Google Developer Documentation Style Guide (https://developers.google.com/style)
```

Reply:
Changed "Google's Developer docs style guide" to "Google Developer Documentation Style Guide" for clarity and consistency with the source.